### ISSUE TITLE
Make $selfSubUrl in Schema optional

### ISSUE THREAD

**Issue by joshbodine21** [neomerx/json-api#104]
I have been scouring the documentation and cannot seem to figure out how to get the "Included" section to show up with the attributes of my relationships. The relationships show up fine in the "data" section with the appropriate ID and type, but the "Included" section never displays no matter what I try?

**Comment by neomerx** [comment]
https://github.com/neomerx/json-api/wiki/Schemas search for `getIncludePaths`

**Comment by joshbodine21** [comment]
This was already implemented in my code but still didn't work, so I figured I was doing something wrong:

```
protected $resourceType = 'Accounts';
protected $selfSubUrl  = '/Accounts/';

public function getId($Account)
{
    return $Account->RecordID;
}

public function getAttributes($Account)
{
    /** @var Author $author */
    return [
        'Name' => $Account->Name,
        'Revenue' => $Account->Revenue,
        'URL' => $Account->URL,
        'IsDeleted' => (bool)$Account->IsDeleted,
    ];
}


public function getRelationships($Account, array $includeRelationships = []) {
    return [
        'Status' => [self::DATA => $Account->Status],
        'Type' => [self::DATA => $Account->Type],
        'Territory' => [self::DATA => $Account->Territory],
        'Industry' => [self::DATA => $Account->Industry],
        'EmployeeRange' => [self::DATA => $Account->EmployeeRange],
        'Address' => [self::DATA => $Account->Address->all()],
    ];
}

public function getIncludePaths() {
    return [
        'Account',
        'Account.Status',
        'Account.Type',
        'Account.Territory',
        'Account.Industry',
        'Account.EmployeeRange',
        'Account.Address'
    ];
}
```

Thanks for your help.

**Comment by neomerx** [comment]
I actually have an idea. If `$selfSubUrl` is not specified it will be constructed as

``` php
'/' . $resourceType . '/'
```

**Comment by joshbodine21** [comment]
Not sure what you mean. Are you saying I should NOT specify selfSubUrl and the "Included" section will then show up? I just tried to remove it and I get a schema error telling me that selfSubUrl was not specified?

**Comment by neomerx** [comment]
In the next release `$selfSubUrl` will be optional. If not specified it will be constructed as

``` php
'/' . $resourceType . '/'
```

**Comment by neomerx** [comment]
Released in 0.6.5
